Samples/nuget.config

```diff
@@ -9,10 +9,9 @@
     </packageRestore>
     <packageSources>
         <clear />
-        <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
```

Comment:
What does removing this do to all of the other NuGets in the repo? For example, how other samples are bringing in WinAppSDK?